### PR TITLE
React to runtime renaming

### PR DESCRIPTION
**Pull request by ChengTian**
Reacting to https://github.com/aspnet/Coherence-Signed/issues/27, https://github.com/aspnet/Coherence-Signed/issues/31 and https://github.com/aspnet/Coherence-Signed/issues/21

@Eilon @muratg @anurse @davidfowl

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -28,11 +28,11 @@ if test ! -d packages/KoreBuild; then
 fi
 
 if ! type k > /dev/null 2>&1; then
```
Comment (muratg) [inline build.sh]: Does "type k" need to change?

Comment (davidfowl) [inline build.sh]: Yep

**Comment by muratg** [comment]
Apart from "type k" feedback, looks good to me.  I'd say :shipit: for now.

File: build.sh [hunk 2]
```diff
@@ -27,12 +27,12 @@ if test ! -d packages/KoreBuild; then
     mono .nuget/nuget.exe install Sake -version 0.2 -o packages -ExcludeVersion
 fi
 
-if ! type k > /dev/null 2>&1; then
```
Comment (ChengTian) [inline build.sh:30]: Fixed

**Pull request merged**